### PR TITLE
Force upgrade packages again.

### PR DESCRIPTION
**Pull request by digiter**
## Description

Retry updating the packages.

## Related Issues

#40989

## Tests

N/A

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (Please read [Handling breaking changes]). *Replace this with a link to the e-mail where you asked for input on this proposed change.*
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

**Comment by digiter** [comment]
Hi @jonahwilliams , do you know how to fix web_tests-linux?

**Comment by jonahwilliams** [comment]
You need to pin all of the build_runner dependencies

File: packages/flutter_tools/pubspec.yaml [hunk 1]
```diff
@@ -46,32 +46,29 @@ dependencies:
   test_core: 0.2.5
 
   # Code generation dependencies
-  build_runner_core: 3.1.1
+  build_runner_core: 4.0.0
```
Comment (jonahwilliams) [inline packages/flutter_tools/pubspec.yaml]: See: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/commands/update_packages.dart#L22

**Review by jonahwilliams — CHANGES_REQUESTED** [review]

File: packages/flutter_tools/pubspec.yaml (hunk 1 above)
Comment (jonahwilliams) [reply packages/flutter_tools/pubspec.yaml]: This needs to include all of the build_* versions

Comment (digiter) [reply packages/flutter_tools/pubspec.yaml]: That works! Please take another look on the new changes.

**Review by zanderso — APPROVED** [review]

**Pull request merged**